### PR TITLE
basic routes for usage

### PR DESCRIPTION
**Pull request by jamesgorrie**
__WIP__

Adding the basic usage routes so we know what the shape of the API should be.

### CODE REVIEW

File: usage/conf/routes [hunk 1]
```diff
@@ -1,5 +1,9 @@
 GET     /                                               controllers.UsageApi.index
 
+GET     /usage/image/:mediaId                           controllers.UsageApi.forMedia(mediaId: String)
```
Comment (jamesgorrie) [inline usage/conf/routes]: For the Grid

File: usage/conf/routes [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+GET     /usage/article/:articleId                       controllers.UsageApi.forArticle(articleId: String)
+GET     /usage/date/:date                               controllers.UsageApi.forDate(date: String)
```
Comment (jamesgorrie) [inline usage/conf/routes]: For RCS

Comment (jamesgorrie) [inline usage/conf/routes]: THis could probably be an aggregation of CAPI vs this API unless we want print in it?

**Comment by kenoir** [comment]
Good way to make sure we meet in the middle is trying to put usage records into the DB via the API.

**Comment by jamesgorrie** [comment]
^ Explain more to me on Monday

File: usage/conf/routes [hunk 3]
```diff
@@ -1,5 +1,8 @@
 GET     /                                               controllers.UsageApi.index
 
+GET     /usage/media/:mediaId                           controllers.UsageApi.forMedia(mediaId: String)
+GET     /usage/content/:contentId                       controllers.UsageApi.forContent(contentId: String)
```
Comment (akash1810) [inline usage/conf/routes]: What's this route for?

Comment (jamesgorrie) [inline usage/conf/routes]: TO query via a content ID or a media ID. We need to gather more requirements for this.

**Comment by kenoir** [comment]
Merged into capi-poller branch.

**Pull request closed without merging**